### ISSUE TITLE
Option to export to BibItem[Feature Request] 

### ISSUE THREAD

**Issue by mohammedsaif-fb** [Future-Scholars/paperlib#458]
Option to export to BibItem

**Comment by GeoffreyChen777** [comment]
something like this?

```
\bibitem{texbook}
Donald E. Knuth (1986) \emph{The \TeX{} Book}, Addison-Wesley Professional.
```

Currently you can export bibtex and transform it by https://asouqi.github.io/bibtex-converter/

**Comment by mohammedsaif-fb** [comment]
Yes , I too use the tool , it would be nice if it is natively available

**Comment by GeoffreyChen777** [comment]
Sure, we will try to implement it.

**Comment by GeoffreyChen777** [comment]
Implemented in #463 .
